kuma/static/js/components/account-management/delete-user-confirmation-button.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+    'use strict';
+
+    var deleteUserConfirm = document.getElementById('delete-user-confirm');
+    var deleteUserForm = document.getElementById('delete-user-form');
+
+    if (!deleteUserForm) {
+        return;
+    }
+
+    /* once a user select one of the attribution choices,
+        enable the submit button */
+    deleteUserForm.addEventListener('change', function(event) {
+        if (event.target.checked) {
+            deleteUserConfirm.classList.remove('disabled');
+            deleteUserConfirm.classList.add('negative');
+            deleteUserConfirm.removeAttribute('disabled');
```

Comment:
The button, when it's disabled, doesn't look like it's disabled. Not sure I care though :)

Reply:
Really? Looks disabled here https://github.com/mdn/kuma/pull/6378#issuecomment-582051746 and there is also no little hand when you hover over it. In fact, it does not even have a hover state or respond to pointer events.